### ISSUE TITLE
Supporting stand-alone Geocoder without map ?

### ISSUE THREAD

**Issue by Steven-Fabre** [SamSamskies/react-map-gl-geocoder#130]
Hello, I'd like to know if it's gonna be able to run alone without displaying a map like in Vanilla JS ? ( https://docs.mapbox.com/mapbox-gl-js/example/mapbox-gl-geocoder-no-map/ )

**Comment by SamSamskies** [comment]
Nah. It's only meant to be used with `react-map-gl`. Probably not that hard to copy pasta the code and modify it for that purpose though. All the code is in one file https://github.com/SamSamskies/react-map-gl-geocoder/blob/master/src/index.js.
